plugins/ccgx/fu-ccgx-hpi-device.c

```diff
@@ -1380,8 +1378,14 @@ fu_ccgx_hpi_device_setup (FuDevice *device, GError **error)
 	}
 
 	/* set name to be more descriptive */
-	name = g_strdup_printf ("USB-I2C Bridge (%s)", fu_ccgx_hpi_device_get_name (self));
+	if (self->device_name) 
+		name = g_strdup_printf ("%s (%s)", self->device_name, fu_ccgx_hpi_device_get_fw_name (self));
+	else
+		name = g_strdup_printf ("USB-I2C Bridge (%s)", fu_ccgx_hpi_device_get_fw_name (self));
 	fu_device_set_name (FU_DEVICE (self), name);
+	
+	summary = g_strdup_printf ("PD-IC %s Firmware", fu_ccgx_hpi_device_get_fw_name (self));
```

Comment:
It seems a bit odd to me to use the FW name both in the title and the summary keys.  I think that one place is enough (and in my opinion summary makes more sense)

Reply:
fixed at [https://github.com/fwupd/fwupd/pull/2013/commits/d40f67f4c8af3ebfc2342f155d367eb7b6b73e15](url)